x/ecocredit/server/server.go

```diff
@@ -22,23 +22,23 @@ const (
 	ClassInfoTablePrefix     byte = 0x5
 	BatchInfoTablePrefix     byte = 0x6
 
-	ProjectInfoTablePrefix    byte = 0x12
-	ProjectInfoTableSeqPrefix byte = 0x13
-	ProjectsByClassIDIndex    byte = 0x14
-	BatchesByProjectIndex     byte = 0x15
+	ProjectInfoTablePrefix    byte = 0x10
+	ProjectInfoTableSeqPrefix byte = 0x11
+	ProjectsByClassIDIndex    byte = 0x12
+	BatchesByProjectIndex     byte = 0x13
```

Comment:
I don't think this file should have changed in this PR. Looks like something got messed up

Reply:
This prevents the sim tests to pass.

Reply:
basically there is a conflict between the prefixes.

Reply:
this got broken in https://github.com/regen-network/regen-ledger/pull/647